src/test/addrman_tests.cpp

```diff
@@ -48,7 +48,7 @@ class CAddrManCorrupted : public CAddrManSerializationMock
         unsigned char nVersion = 1;
         s << nVersion;
         s << ((unsigned char)32);
-        s << nKey;
+        s << ((uint256)1);
```

Comment:
I slightly prefer `{}` constructors (`uint256{1}`) or `uint256::ONE`.

Reply:
Yes, I agree that's better. Changed to `uint256::ONE`.